docs/GettingStarted.md

```diff
@@ -1526,11 +1526,16 @@ run app
 | `quantize.query` | Hash containing options for query portion of URL quantization. May include `:show` or `:exclude`. See options below. Option must be nested inside the `quantize` option. | `{}` |
 | `quantize.query.show` | Defines which values should always be shown. Shows no values by default. May be an Array of strings, or `:all` to show all values. Option must be nested inside the `query` option. | `nil` |
 | `quantize.query.exclude` | Defines which values should be removed entirely. Excludes nothing by default. May be an Array of strings, or `:all` to remove the query string entirely. Option must be nested inside the `query` option. | `nil` |
+| `quantize.query.obfuscate` | Defines query string redaction behaviour. Redacts nothing by default. May be a hash of options, or `:internal` to use the default internal obfuscation settings. Note that obfuscation is a string-wise operation, not a key-value operation. When enabled, `query.show` defaults to `:all` if otherwise unset. Option must be nested inside the `query` option. | `nil` |
```

Comment:
How does this default of `nil` work with the default for `quantize.query.obfuscate.regex` which is `:internal`?

Also, how can a user completely disable all obfuscation (they trust all their strings safe to flush and low cardinality)?

Reply:
`nil` disables obfuscation, preserving the current behaviour as a default.

I mimicked `quantize.query.show` here, where the default is `nil` too, and whose effect is inferred from the "does X by default" sentence. Do you think we should we update the "May be a ..." clauses to include the default values, and remove the "does X by default" sentence? (which is then implied by the default value column).

Reply:
> Do you think we should we update the "May be a ..." clauses to include the default values, and remove the "does X by default" sentence? (which is then implied by the default value column).

I think this would clarify it for me 👍